common/terms.html

```diff
@@ -79,7 +79,19 @@
     <dd>
       <p>Translated to platform-specific <a class="termref" data-lt="accessibility api">accessibility APIs</a> as defined in the <a href="" class="core-mapping">Core Accessibility API Mappings</a>.</p>
     </dd>
-	<dt><dfn data-lt="graphical document|graphical documents">Graphical Document</dfn></dt>
+  </dd>
+  <dt><dfn>Focusable</dfn></dt>
+  <dd>
+    <p>An element or an area in a document is focusable, if it can get the system focus. In different circumstances one can use one or more of following methods to set focus:</p>
```

Comment:
To remain consistent with other references to "focus", I would suggest "receive focus" rather than "get". For examples where focus is referenced:

- https://w3c.github.io/aria/#alert
- https://w3c.github.io/aria/#status
- https://w3c.github.io/aria/#aria-disabled